aws-predictions/src/main/java/com/amplifyframework/predictions/aws/adapter/RekognitionResultTransformers.java

```diff
@@ -127,13 +130,17 @@ public static AgeRange fromRekognitionAgeRange(@Nullable aws.sdk.kotlin.services
      */
     @Nullable
     public static IdentifiedText fromTextDetection(@Nullable TextDetection text) {
-        if (text == null) {
+        if (text == null || text.getDetectedText() == null || text.getConfidence() == null) {
             return null;
         }
+        RectF box = null;
+        if (text.getGeometry() != null) {
+            box = fromBoundingBox(text.getGeometry().getBoundingBox());
+        }
         return IdentifiedText.builder()
                 .text(text.getDetectedText())
                 .confidence(text.getConfidence())
-                .box(fromBoundingBox(text.getGeometry().getBoundingBox()))
+                .box(box)
```

Comment:
is it okay if this final `box` is `null`?  I'd expect an `if(box == null) return null;` and likewise throughout the PR

Reply:
sounds good!